docs/variants/talos_2/releases.md

```diff
@@ -39,18 +39,26 @@ For details about our release process please read
 
 ### Binaries
 
-[raptor-cs_talos-2_bootblock_v0.5.0.signed.ecc](https://3mdeb.com/open-source-firmware/Dasharo/raptor-cs_talos-2/v0.5.0/raptor-cs_talos-2_bootblock_v0.5.0.signed.ecc){ .md-button }
-[sha256](https://3mdeb.com/open-source-firmware/Dasharo/raptor-cs_talos-2/v0.5.0/raptor-cs_talos-2_bootblock_v0.5.0.signed.ecc.sha256){ .md-button }
-[sha256.sig](https://3mdeb.com/open-source-firmware/Dasharo/raptor-cs_talos-2/v0.5.0/raptor-cs_talos-2_bootblock_v0.5.0.signed.ecc.sha256.sig){ .md-button }
```

Comment:
@krystian-hebel are you using pre-commit? Why you changed the way binaries are linked? Why you changed the name?

Reply:
I wasn't at that time using pre-commit. This whole section is auto-generated by `dasharo-templates`, I haven't changed anything here manually AFAICT.

Reply:
We need to fix that in template